packages/data-broker/src/services/dhis/builders/addTrackedEntityCodes.js

```diff
@@ -0,0 +1,3 @@
+export const addTrackedEntityCodes = async (models, events) => {
+  return events;
+};
```

Comment:
Unused?

Reply:
Ah didn't spot this in my once-over, removed